pyro/distributions/torch.py

```diff
@@ -101,6 +101,17 @@ def log_prob(self, value):
         return (-value - 1) * torch.nn.functional.softplus(self.logits) + self.logits
 
 
+class LogNormal(torch.distributions.LogNormal, TorchDistributionMixin):
+    def __init__(self, loc, scale, validate_args=None):
+        base_dist = Normal(loc, scale)
```

Comment:
Can you add a comment that

```py
# This differs from torch.distributions.LogNormal only in that base_dist is
# a pyro.distributions.Normal rather than a torch.distributions.Normal.
```